build.gradle

```diff
@@ -130,7 +130,7 @@ sonarqube {
   properties {
     property "sonar.projectName", "Reform :: et-sya-api"
     property "sonar.projectKey", "uk.gov.hmcts.reform:et-sya-api"
-    property "sonar.coverage.exclusions", "**/gov/hmcts/reform/et.syaapi/config/**"
+    property "sonar.coverage.exclusions", "**/gov/hmcts/reform/et/syaapi/config/OpenAPIConfiguration.java"
```

Comment:
Can we keep **/gov/hmcts/reform/et/syaapi/config/** here?

Reply:
Ok update to exclude all of config.